impove-katacoda-scenario (#12)
kubernetes-sigs/contributor-katacoda: Scenario-setup-linux/step1.md

> As Kubernetes is a large project with multiple contributors, in order to make any changes to this project, we need to create a copy of repository using fork. `Forking` helps to create a copy of the repository and make changes in your own branch/repository without affecting the upstream repository.

Reviewer: ```suggestion
As Kubernetes is a large project with multiple contributors, in order to make any changes to this project, we need to create a copy of the repository by forking it.
```
